git/00.setup.md

```diff
@@ -19,30 +19,6 @@ brew install hub
 alias git="hub"
 ```
 
-## Additional config
-
-It is recommended set the follow configurations for git to make your life
-easier.
-
-Set your default push to tracking. Avoid having to type extra characters and
-prevent accidentally pushing feature branch to master it recommended to set your
-default push setting to tracking using the following.
-
-```bash
-git config --global push.default tracking
-```
-
-Next time you want to push a just run `git push`, if you haven't pushed the
```

Comment:
Yep! I actually don't think that check is a bad idea in terms of a recommendation. Plus it includes the command to use to resolve it. But I don't really care either way. What do you think? We could mention it as a possibility I suppose.